app/api/contests/get_group_by_name.go

```diff
@@ -122,11 +122,11 @@ func (srv *Service) getGroupByName(w http.ResponseWriter, r *http.Request) servi
 			Joins(`
 				LEFT JOIN `+"`groups`"+` AS user_group
 					ON user_group.id = groups_groups_active.child_group_id AND user_group.type = 'User' AND
-						user_group.name LIKE ?`, groupName).
+						user_group.name = ? AND LENGTH(user_group.name) = LENGTH(?)`, groupName, groupName).
```

Comment:
I don't really get what you are trying to avoid. 
If you want exact matching, why do you check length in addition? Why having used 'like' in the first place?

Reply:
`LIKE` was used by mistake here. The `LENGTH()` is needed because MySQL stips trailing spaces in comparison for non-binary strings (see https://dev.mysql.com/doc/refman/8.0/en/charset-binary-collations.html#charset-binary-collations-trailing-space-comparisons). We could use conversion to BINARY type here, but doing so would make the comparison case-sensitive.